pkg/sdk/testint/materialized_views_gen_integration_test.go

```diff
@@ -326,7 +326,7 @@ func TestInt_MaterializedViews(t *testing.T) {
 		err := client.Views.Alter(ctx, alterRequestSetTags)
 		require.NoError(t, err)
 
-		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), id, sdk.ObjectTypeTable)
+		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), id, sdk.ObjectTypeMaterializedView)
```

Comment:
why in some tests we check for err nil and in some we also validate the tag's value?

Reply:
That's because when a tag is not set, GetTag returns an error. I'd like to change this, so that `nil` is returned as a tag value in the next PR.

Reply:
Oky (keeping it open then)